tests/ci/docker_images/build_cryptofuzz_modules.sh

```diff
@@ -12,10 +12,8 @@ export CXXFLAGS="-fsanitize=address,undefined,fuzzer-no-link -D_GLIBCXX_DEBUG -O
 # Setup base of Cryptofuzz
 cd "$FUZZ_ROOT"
 MODULES_ROOT="${FUZZ_ROOT}/modules"
-#git clone --depth 1 https://github.com/guidovranken/cryptofuzz.git
-git clone https://github.com/guidovranken/cryptofuzz.git
+git clone --depth 1 https://github.com/guidovranken/cryptofuzz.git
 cd cryptofuzz
-git checkout 76ffeff944403cdd840f06b8fc42e131e6258f36
 git rev-parse HEAD
```

Comment:
Nit:
> Cryptofuzz previously required changes from BoringSSL not yet merged into AWS-LC. With the recent merge we can use the latest version again.

The latest cryptofuzz may bring new variants.  Should the commit be locked?